semgrep_output_v1.atd

```diff
@@ -1824,22 +1824,33 @@ type output_format = [
   | Emacs
 ]
 
-(* TODO: merge with Manifest_kind.ml *)
-type manifest_kind <python decorator="dataclass(frozen=True)"> = [
-  | RequirementsIn (* Manifest file for pip *)
+type manifest_kind 
```

Comment:
@mjambon what do we need to add to generate proper Variant like RequirementsIn instead of the `RequirementsIn in the generated OCaml code?

Reply:
Hmm, it looks like that might even be supported today with `<ocaml repr="classic">`: https://atd.readthedocs.io/en/latest/atdgen.html#sum-types

Do you think I should switch this one over?

Reply:
I prefer yes. They are simpler to handle.